kibbeh/src/types/Gist.ts

```diff
@@ -0,0 +1,65 @@
+export type Gist = {
+  url: string;
+  forks_url: string;
+  commits_url: string;
+  id: string;
+  node_id: string;
+  git_pull_url: string;
+  git_push_url: string;
+  html_url: string;
+  files: { [key: string]: File };
+  public: true;
+  created_at: string;
+  updated_at: string;
+  description: string;
+  comments: number;
+  user: null;
+  comments_url: string;
+  owner: User;
+  forks: [];
```

Comment:
what

Reply:
I wrote the types myself, because I couldn't find them anywhere else. I don't think we're gonna need to list the forks of a Gist.

Reply:
just remove the property + im pretty sure `@octokit/types` has those types already